pkg/config/aliases_darwin.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
+)
+
+// Aliases to setup package
```

Comment:
What is the reason of aliases? Why not renaming `config.DefaultSecurityAgentLogFile` to `pkgconfigsetup.DefaultSecurityAgentLogFile`

Reply:
We introduced aliases to limit the code changes during the modularization work. We have a follow-up, Jira, to get rid of the aliases. The follow-up PRs should be simple.